### PR TITLE
feature/144_url_shorten

### PR DESCRIPTION
**Pull request by astrochun**
See issue #144
﻿

### CODE REVIEW

**Review by astrochun — COMMENTED** [review]
Changes look good.

Edge case that doesn't need to be address now: If surname is too long, TinyURL will eventually fail and throw a `HTTPError`

**Review by damian-romero — COMMENTED** [review]
The code looks good. I'll need to test it tomorrow before approving unless the merge is urgent.

**Review by damian-romero — APPROVED** [review]
- Tested with deposit `12904412`. Works like a charm. 
- Did not test for edge case with longer depositor's name.

**Pull request merged**
